xbmc/Application.cpp

```diff
@@ -4085,16 +4085,24 @@ PlayBackRet CApplication::PlayFile(const CFileItem& item, bool bRestart)
       CJobManager::GetInstance().PauseJobs();
     }
 
-    // don't hold graphicscontext here since player
-    // may wait on another thread, that requires gfx
-    CSingleExit ex(g_graphicsContext);
+    if (item.Exists()) // this is also an implicit call to CWakeOnAccess::WakeUpHost() if enabled
+    {
+      // don't hold graphicscontext here since player
+      // may wait on another thread, that requires gfx
+      CSingleExit ex(g_graphicsContext);
 
-    iResult = m_pPlayer->OpenFile(item, options);
+      iResult = m_pPlayer->OpenFile(item, options);
+    }
+    else
+    {
+      CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Error, 
```

Comment:
If this gets called from here 
https://github.com/xbmc/xbmc/blob/943c92e494ffd35dab79eed810b15f5fb779f342/xbmc/PlayListPlayer.cpp#L287

And fails it will show a toast and a popup with the same messages, due to this:
https://github.com/xbmc/xbmc/blob/943c92e494ffd35dab79eed810b15f5fb779f342/xbmc/PlayListPlayer.cpp#L305

Not saying what your doing here is wrong, but we might want to check all places that are calling this and acting on the return value..